go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/influxdata/telegraf
 go 1.16
 
 require (
-	cloud.google.com/go v0.56.0
-	cloud.google.com/go/bigquery v1.4.0
-	cloud.google.com/go/pubsub v1.2.0
+	cloud.google.com/go v0.72.0
+	cloud.google.com/go/bigquery v1.8.0
+	cloud.google.com/go/pubsub v1.3.1
```

Comment:
Are all of those package upgrades necessary or did they make it in by accident? Just wondering as it seems a massive bunch... Same for all updates below.

Reply:
I reverted go.mod and started fresh, looks much better now!